pkgs/development/libraries/xine-lib/default.nix

```diff
@@ -38,6 +30,6 @@ stdenv.mkDerivation rec {
     homepage = "http://www.xine-project.org/";
```

Comment:
```suggestion
    homepage = "https://sourceforge.net/projects/xine/";
```

Reply:
Actually I think http://xine.sourceforge.net/home is more homepage-like.